perl/p5-mail-spamassassin/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           perl5 1.0
 
 perl5.branches      5.26
-perl5.setup         Mail-SpamAssassin 3.4.1
+perl5.setup         Mail-SpamAssassin 3.4.2
 revision            5
```

Comment:
Remove the `revision` line because it starts at 0 when increasing the version and that is the default value.